scripts/zip_docker_logs.sh

```diff
@@ -0,0 +1,42 @@
+#/bin/bash
+
+##############################################3
+# This script gathers logs for ALL containers
+# running and outputs them into a zip file in
+# your home directory here:
+#
+#       ~/.medic/cht-docker/support_logs
```

Comment:
```suggestion
# ~/.cht/docker/support_logs
```

Maybe we'd want to keep "medic" out of it.

Reply:
Ah - yeah - I can see that.  I was leveraging the existing [directory CHT Docker Helper creates](https://github.com/medic/cht-core/blob/master/scripts/docker-helper-4.x/cht-docker-compose.sh#L68).  You think it's worth keeping these the same or diverging them?

Reply:
I think keeping the same makes sense, though I do think we should eventually remove the "medic" out of it. Do you see a bunch of "Canonical" folders in your Ubuntu install?